src/eme.js

```diff
@@ -463,7 +451,7 @@ export const standard5July2016 = ({
       );
 
       getLicenseFn = promisifyGetLicense(keySystem, getLicense, eventBus);
-      getContentIdFn = getContentId;
+      contentId = getContentId ? getContentId(initData) : null;
```

Comment:
We'll also have to do it above in `addPendingSessions`. It might be worth trying to do it in `standardizeKeySystemOptions`

Reply:
I don't think it makes sense at that point, We now do it in the just before `addSession`